public/js/views/work.tpl

```diff
@@ -64,9 +64,16 @@
 				<div class='locations col-sm-4'>
 					<div class='uibox'>
 						<div class="title">Locations</div>
+						<div class="world-map"></div>
+						<table class="device-stats">
+							<tr width="70%"><th>Location</th><th>Devices</th></tr>
```

Comment:
That would make much more sense :) But as we're using Bootstrap it's
probably even better to use its styles (col-sm-8 and col-sm-4 for the
cells, will probably also require class="table" and proper tbody added to
the table)

On Fri, May 22, 2015 at 4:42 PM cPu1 <notifications@github.com> wrote:

> In public/js/views/work.tpl
> <https://github.com/pcarranzav/parallellocalypse-hub/pull/5#discussion_r30895479>
> :
>
> > @@ -64,9 +64,16 @@
> >  				<div class='locations col-sm-4'>
> >  					<div class='uibox'>
> >  						<div class="title">Locations</div>
> > +						<div class="world-map"></div>
> > +						<table class="device-stats">
> > +							<tr width="70%"><th>Location</th><th>Devices</th></tr>
>
> That's wrong. I intended to put it to the first cell I guess.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pcarranzav/parallellocalypse-hub/pull/5/files#r30895479>
> .
>